src/users/cd-login-or-register.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="cd-login-or-register">
+    <createAccount ref="bookingCreateAccountRef" :context="context" @registered="next()"></createAccount>
+    <div class="cd-login-or-register__login">
+      <redirectToLogin :url="redirectionUrl"></redirectToLogin>
+    </div>
+  </div>
+</template>
+<script>
+  import StoreService from '@/store/store-service';
+  import EventService from '@/events/service';
+  import DojoService from '@/dojos/service';
+  import CreateAccount from '@/users/cd-create-account';
+  import RedirectToLogin from '@/users/cd-redirect-to-login';
+
+
+  export default {
+    name: 'LoginOrRegister',
+    props: ['eventId'],
+    components: {
+      CreateAccount,
+      RedirectToLogin,
+    },
+    data() {
+      return {
+        eventDetails: {},
+        dojoDetails: {},
+      };
+    },
+    methods: {
+      async loadEvent() {
+        this.eventDetails = (await EventService.loadEvent(this.eventId)).body;
+      },
+      async loadDojo() {
+        this.dojoDetails = (await DojoService.getDojoById(this.eventDetails.dojoId)).body;
+      },
+      next() {
+        StoreService.save('selected-event', this.eventDetails);
```

Comment:
Do we actually need this anymore? Pretty sure we can dump StoreService altogether now, which I would be happy to do.

Reply:
I'm still using it on ticket selection, but it makes very few sense imho : the info is in the url.
If we wanted to make it clean, using a store/vuex would be more than enough ? The reason for the store-service was local storage, but i find it unecessary for a 1 form page